src/components/Terminal.jsx

```diff
@@ -287,48 +295,48 @@ export default class Terminal extends React.Component {
       },
       prompt: {
         ...sourceStyles.prompt,
-        color: this.props.promptLabelColor || '#EE9C34'
+        color: this.props.promptLabelColor
       },
       input: {
         ...sourceStyles.input,
-        color: this.props.promptTextColor || '#F0BF81',
+        color: this.props.promptTextColor,
         fontFamily: this.props.inputFontFamily || 'monospace'
       }
     }
 
     return (
       <div
         ref={this.terminalRoot}
-        className={this.props.className || null}
+        className={this.props.className}
         name={'react-console-emulator'}
         style={styles.container}
         onClick={this.focusTerminal}
       >
         <div
-          className={this.props.contentClassName || null}
+          className={this.props.contentClassName}
           name={'react-console-emulator__content'}
           style={styles.content}
         >
           {this.getStdout()}
           <div
-            className={this.props.inputAreaClassName || null}
+            className={this.props.inputAreaClassName}
             name={'react-console-emulator__inputArea'}
             style={styles.inputArea}
           >
-            <span
-              className={this.props.promptLabelClassName || null}
-              style={styles.prompt}
-            >
-              {this.props.promptLabel || '$'}
-            </span>
+            {
+              this.props.promptLabel
+                ? this.props.promptLabel()
```

Comment:
There's a smarter way to go about this. See my comment further down for nore details.